todo_app/data/trello_items.py

```diff
@@ -0,0 +1,164 @@
+from requests import request
+from os import environ
+from todo_app.data.Item import Item
+
+TRELLO_KEY = environ["TRELLO_KEY"]
+TRELLO_TOKEN = environ["TRELLO_TOKEN"]
+TRELLO_BOARD_ID = environ["TRELLO_BOARD_ID"]
```

Comment:
Nothing wrong with this, but note that it may catch you out later when you look at testing due to the order in which things happen (e.g. this runs when the file is imported, so you might incorrectly set these before the test has a chance to put fake variables into the environment) - so it might be worth pre-emptively moving to load these at run time (e.g. by switching any use of the variables to load fresh) e.g. line 17 would be:
```
url= f"https://api.trello.com/1/boards/{environ["TRELLO_BOARD_ID"]}/lists"
```